src/pages/FirstRun.vue

```diff
@@ -3,46 +3,44 @@
     <h2 data-test="k8s-settings-header">
       Welcome to Rancher Desktop
     </h2>
-    <div class="k8s-settings">
```

Comment:
My idea with removing the `k8s-settings` div was to make use of the flex container defined in `src/layouts/default.vue`. An alternative approach would be to introduce a new flex container in `FirstRun.vue`